addons/binding/org.openhab.binding.vera/src/main/java/org/openhab/binding/vera/internal/converter/VeraDeviceStateConverter.java

```diff
@@ -0,0 +1,151 @@
+
+        String channelType = channel.getUID().getId().split("-")[0];
+        if (BATTERY_CHANNEL.equals(channelType)) {
+            return getMultilevelState(device.getBatterylevel());
+        }
+        if (SENSOR_METER_KWH_CHANNEL.equals(channelType)) {
+            return getMultilevelState(device.getKwh());
+        }
+        if (SENSOR_METER_W_CHANNEL.equals(channelType)) {
+            return getMultilevelState(device.getWatts());
+        }
+
+        int subcategory = Integer.parseInt(device.getSubcategory());
+        switch (device.getCategoryType()) {
+            case Controller:
+            case Interface:
+                break;
+            case DimmableLight:
+                switch (subcategory) {
+                    case 1:
+                    case 2:
+                    case 3:
+                        return getPercentState(device.getLevel());
+                    case 4:
+                        return getPercentState(device.getLevel());// TODO getColorState(device.light);
+                }
+                break;
+            case Switch:
+                return getBinaryState(device.getStatus());
+            case SecuritySensor:
+                switch (subcategory) {
+                    case 1:
+                        return getDoorlockState("1".equals(device.getTripped()) ? "0" : "1");
+                    default:
+                        return getBinaryState(device.getTripped());
+                }
+            case HVAC: // TODO
+                logger.warn("TODO: {}, {}", device, device.getCategoryType());
+                break;
+            case DoorLock:
+                return getBinaryState(device.getLocked());
+            case WindowCovering:
+                return getPercentState(device.getLevel());
+            case GenericSensor:
+                return getMultilevelState(device.getLevel());
+            case SceneController:
+                return getBinaryState(device.getStatus());
+            case HumiditySensor:
+                return getMultilevelState(device.getHumidity());
+            case TemperatureSensor:
+                return getMultilevelState(device.getTemperature());
+            case LightSensor:
+                return getMultilevelState(device.getLight());
+            case PowerMeter:
+                return getMultilevelState(device.getLevel());
+            case UVSensor:
+                return getMultilevelState(device.getLevel());
+            case Camera:
+            case AV:
+            case ZWaveInterface:
+            case InsteonInterface:
+            case RemoteControl:
+            case IRTransmitter:
+            case GenericIO:
+            case SerialPort:
+            case AlarmPanel:
+            case AlarmPartition:
+            case Siren:
+            case Weather:
+            case PhilipsController:
+            case Appliance:
+                logger.warn("TODO: {}, {}", device, device.getCategoryType());

```

Comment:
Can you change it to `debug` instead? I wouldn't like a log file full of warnings just because devices aren't yet supported by a binding. It would also help to make a note about this in the docs.